spec/locale.html

```diff
@@ -277,6 +277,18 @@ <h1>get Intl.Locale.prototype.collation</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-Intl.Locale.prototype.getVariants">
+      <h1>Intl.Locale.prototype.getVariants ( )</h1>
+
+      <emu-alg>
+        1. Let _loc_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_loc_, [[InitializedLocale]]).
+        1. Let _variantsString_ be GetLocaleVariants(_loc_.[[Locale]]).
```

Comment:
This can be `undefined`, which is not supported by `StringSplitToList()`.
```suggestion
        1. Let _variantsString_ be GetLocaleVariants(_loc_.[[Locale]]).
        1. If _variantsString_ is *undefined*, then
          1. Return ! ArrayCreate(0).
```